### ISSUE TITLE
Placeholder text is not getting cleared when text is entered

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#596]
## Expected behaviour 
When text is entered into the text box, place holder text should not get appended.

## Actual behaviour 
The text is getting appended to the placeholder

## Steps to replicate 
* Open the REPL
```
> openBrowser()
 ✔ Browser opened
> goto("cleartrip.com")
 ✔ Navigated to url http://cleartrip.com
> write("Bangaluru",into(textBox(below("From"))))
 ✔ Wrote Bangaluru into the text field Below From
```
> However the text Bengaluru is getting appended to `Any worldwide city or airport`. This is the place holder of the textBox

**Version**
commit - d4a99381515a3a6262f829eee79a8e00197ada05

**Comment by NivedhaSenthil** [comment]
Fix available in 9d926a8

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit verison ```9d926a8 ```
